packages/cra-template-typescript/template/gitignore

```diff
@@ -5,6 +5,9 @@
 /.pnp
 .pnp.js
 
+# build cache info
+tsconfig.tsbuildinfo
+
```

Comment:
We should set compilerOptions `tsBuildInfoFile: paths.cacheTsBuildInfoFile` in the `ForkTsCheckerWebpackPlugin`